app/uk/gov/hmrc/fhregistrationfrontend/views/enrolment_pending.scala.html

```diff
@@ -15,47 +15,57 @@
  *@
 
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
 
-@this(
-    mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
+@this(layout: layout, viewHelpers: ViewHelpers)
 
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-@stylesheets = {<link rel="stylesheet" href='@routes.Assets.at("stylesheets/status.css")'/>}
 
-@title = @{
-  Messages(s"fh.status.received.title")
+@title1 = @{
+  Messages("fh.status.received.title")
 }
 
-@mainTemplate(title = title, linkElement = Some(stylesheets)) {
+@layout(title = title1) {
 
-  <div class="govuk-box-highlight appstatus">
-    <h1 class="heading-xlarge">
-      @title
-    </h1>
-  </div>
+    @viewHelpers.govukPanel(Panel(
+        title = Text(Messages("fh.status.received.title")),
+        content = HtmlContent("")
+    ))
 
-  <h2 class="heading-large">
+    <h2 class="govuk-heading-l">
```

Comment:
if it's h2, then it should be `govuk-heading-m`, else make is h1

Reply:
This should be an h2, not an h1 - there is an h1 in the panel above.

Reply:
@adamliptrot-oc could you please confirm if it's h2 , then should it be a govuk-heading-m or govuk-heading-l?

Reply:
looking at the page , i think it should be govuk-heading-m , but let's wait for confirmation from adam

Reply:
done

Reply:
it's still showing as govuk-heading-l